examples/conversationbot2.py

```diff
@@ -110,29 +111,29 @@ def main():
 
     # Add conversation handler with the states GENDER, PHOTO, LOCATION and BIO
     conv_handler = ConversationHandler(
-        entry_points=[CommandHandler('start', start)],
-
-        states={
-            CHOOSING: [RegexHandler('^(Age|Favourite colour|Number of siblings)$',
-                                    regular_choice,
-                                    pass_user_data=True),
-                       RegexHandler('^Something else...$',
-                                    custom_choice),
-                       ],
-
-            TYPING_CHOICE: [MessageHandler(Filters.text,
-                                           regular_choice,
-                                           pass_user_data=True),
-                            ],
-
-            TYPING_REPLY: [MessageHandler(Filters.text,
-                                          received_information,
-                                          pass_user_data=True),
+            entry_points=[CommandHandler('start', start)],
+
+            states={
+                CHOOSING: [RegexHandler('^(Age|Favourite colour|Number of siblings)$',
+                                        regular_choice,
+                                        pass_user_data=True),
+                           RegexHandler('^Something else...$',
+                                        custom_choice),
                            ],
-        },
 
-        fallbacks=[RegexHandler('^Done$', done, pass_user_data=True)]
-    )
+                TYPING_CHOICE: [MessageHandler(Filters.text,
+                                               regular_choice,
+                                               pass_user_data=True),
+                                ],
+
+                TYPING_REPLY: [MessageHandler(Filters.text,
+                                              received_information,
+                                              pass_user_data=True),
+                               ],
+                },
+
+            fallbacks=[RegexHandler('^Done$', done, pass_user_data=True)]
+            )
```

Comment:
why did you change the indentation here? please revert

Reply:
I moved everything one space forward since a space between the double colons and the Square Brackets. If I remove that one space indent flake8 will complain.